provisioners/apt_proxy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [ "${apt_cache_host:-}" ]
+then
```

Comment:
please move this to the same line as the `if` statement, as-in, `; then`.

Reply:
Additionally I decided to return early. So we get rid of the indentation.